### PR TITLE
Add Kubernetes 1.24 release outline

### PR DESCRIPTION
**Pull request by JamesLaverack**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

Add the initial outline for the Kubernetes 1.24 release cycle, based on the outline from the [release cadence blog post](https://kubernetes.io/blog/2021/07/20/new-kubernetes-release-cadence/#what-s-changing-and-when).

### CODE REVIEW

File: releases/release-1.24/release-team.md
Comment (saschagrunert) [inline releases/release-1.24/release-team.md:1]:
```suggestion
# Kubernetes 1.24 Release Team
```

File: releases/release-1.24/release-team.md
Comment (saschagrunert) [inline releases/release-1.24/release-team.md:17]:
```suggestion
The schedule for all patch releases can be found at [Patch Releases page](/releases/patch-releases.md). It will be updated to include 1.24, once the 1.24 release cycle concludes.
```

File: releases/release-1.24/README.md
Comment (saschagrunert) [inline releases/release-1.24/README.md:41]: I know the [KEP proposes the same](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence#proposal), but how about starting the cycle on January 10th?

**Comment by saschagrunert** [comment]
After this has been merged we should update the KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence

cc @kubernetes/sig-release-leads

File: releases/release-1.24/README.md
Comment (jeremyrickard) [reply releases/release-1.24/README.md:41]: Absolutely agree.

**Comment by jeremyrickard** [comment]
> After this has been merged we should update the KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence
> 
> cc @kubernetes/sig-release-leads

I think we just messed up in the table.  The KEP policy says:

> The first Kubernetes release of a year should start at the second or third week of January to provide people more room after coming back from the Christmas holidays.

Which is what you mentioned above, so we should update the dates in there to reflect. I'll open a PR to do that.

**Comment by JamesLaverack** [comment]
From that KEP:

> This would result in a 15 week release cycle, with 2 weeks between release cycles.

If we started with Monday 10th January 2022 as our Week 1, then the release would be on Tuesday 19th April.

That would mean that 1.25 would have to start two weeks later on Monday 9th May and release on Tuesday 16th August, and 1.26 starts on Monday 5th September and release on Tuesday 13th December. Does that sound about right?

**Comment by saschagrunert** [comment]
> From that KEP:
> 
> > This would result in a 15 week release cycle, with 2 weeks between release cycles.
> 
> If we started with Monday 10th January 2022 as our Week 1, then the release would be on Tuesday 19th April.
> 
> That would mean that 1.25 would have to start two weeks later on Monday 9th May and release on Tuesday 16th August, and 1.26 starts on Monday 5th September and release on Tuesday 13th December. Does that sound about right?

Yep.

**Review by saschagrunert — APPROVED** [review]
/hold 
for EA review
/assign @jrsapi

**Review by jrsapi — APPROVED** [review]
/LGTM

**Comment by JamesLaverack** [comment]
/unhold
Unheld after discussion with @jrsapi offline

File: releases/release-1.24/README.md
Comment (reylejano) [inline releases/release-1.24/README.md:90]: non-blocking nit because this is PR is for the outline and not set official but in the TL;DR v1.24.0 is released on week 15 this suggestion aligns the release week with the v1.24.0 release during week 15.
Also suggest to add publishing the "Deprecations and Removals blog" per 1.22 and 1.23 retros to the timeline.

```suggestion
| Burndown Meetings daily| Lead | **TBC** | week 12 | |
| **Begin [Code Freeze]** (**Time TBC**) | Branch Manager | **TBC** | week 12 | |
| 1.24.0-beta.0 released | Branch Manager | **TBC** | week 12 | |
| Docs deadline - Open placeholder PRs | Docs Lead | **TBC** | week 12 | |
| Deprecations and Removals blog published | Comms | **TBC** | week 12 | |
| **[Test Freeze]** (**Time TBC**) | Branch Manager | **TBC** | week 13 | |
| Docs deadline - PRs ready for review | Docs Lead | **TBC** | week 13 | |
| 1.24.0-rc.0 released | Branch Manager | **TBC** | week 13 | |
| release-1.24 branch created | Branch Manager | **TBC** | week 13 | |
| release-1.24 jobs created | Branch Manager | **TBC** | week 13 | |
| Start final draft of Release Notes | Release Notes Lead | **TBC** | week 13 | |
| Release blog ready to review (**Time TBC**) | Comms / Docs | **TBC** | week 13 | |
| Docs complete - All PRs reviewed and ready to merge | Docs Lead | **TBC** | week 14 | |
| Feature blogs ready to review (**Time TBC**)| Enhancement Owner / SIG Leads | **TBC** | week 14 | |
| Release Notes complete - reviewed & merged to `k/sig-release` (**Time TBC**) | Release Notes Lead | **TBC** | week 14 | |
| 1.24.0-rc.1 released | Branch Manager | **TBC** | week 14 | |
| **v1.24.0 released** | Branch Manager | **TBC** | week 15 | |
| Release blog published | Comms | **TBC** | week 15 | |
| **[Thaw]** | Branch Manager | **TBC** | week 15 | |
| Release retrospective part 2 (**TBC**) | Community | **TBC** | week 16 | |
| Release retrospective part 3 (**TBC**) | Community | **TBC** | week 16 | |
```

**Comment by reylejano** [comment]
My non-blocking nit and [comment](https://github.com/kubernetes/sig-release/pull/1781#discussion_r775664961) can be addressed in a follow-up PR
/lgtm

File: releases/release-1.24/README.md
Comment (JamesLaverack) [reply releases/release-1.24/README.md:90]: This is a great shout. I've made this into commit 5df623d5c8d66ee81dce1adefe467ff149bea0b1 on my follow-up PR https://github.com/kubernetes/sig-release/pull/1798 to add the full schedule.

**Pull request merged**
